smartcontractkit/external-adapters-js
# DAR Cache age update

## Closes [DF-18504](https://smartcontract-it.atlassian.net/browse/DF-18504?atlOrigin=eyJpIjoiYjI1ZTZmZmU5ZTY1NGM1MGFkY2RmNTQyNmU2NzU2OGMiLCJwIjoiaiJ9)

## Description
- Set `CACHE_MAX_AGE` default override to 20 minutes for DAR
- Add debug logging on messages from DAR WS to track price pairs and hit counts



## Steps to Test
1. Build EA
2. Run EA with required credentials and `LOG_LEVEL=debug`
3. Parse the log messages to confirm that prices are streaming accurately. This can also be used to check that low-frequency pairs still hold the last cached price within 20 minutes.

## Quality Assurance



[DF-18504]: https://smartcontract-it.atlassian.net/browse/DF-18504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
